crud/select/filters.lua

```diff
@@ -164,7 +170,7 @@ local function format_value(value)
     elseif type(value) == 'number' then
         return tostring(value)
     elseif type(value) == 'cdata' then
-        return tostring(value)
+        return cdata_formats[value_type] and cdata_formats[value_type](value) or tostring(value)
```

Comment:
Function takes `value_type`, but is uses both `type(value)` and `value_type`. BTW, `box.NULL` is also `cdata`.

Maybe it would be better to [check](https://github.com/tarantool/checks/blob/master/checks.lua#L237) UUID type directly. 

I don't see any reason to separate UUID string formatting - this function is responsible for this:

```lua
elseif utils.is_uuid(value) then
    return ("%q"):format(value)
...
```

Reply:
Added `utils.is_uuid` method, removed `cdata_formats` and `value_type` checking